learn-express-build-strock-trading-platform/server060.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const app = express();
+
+app.listen(3000, function() {
+  console.log('Your app is listening on port 3000.');
+});
+
+app.get('/hi', (req, res) => {
+  res.send('Hi there!');
+});
+
+const prices = {
+  ABC: 48.83,
+  DEF: 2.98,
+  GHI: 3.99,
+  JKL: 99,
+  MNO: 45.38,
+  OPQ: 0.48,
+  RST: 9.32,
+  UVW: 10.94,
+  XYZ: 5.32,
+};
+
+app.get('/buy/:ticker/:shares', (req, res) => {
```

Comment:
Simplify this and bit and break it down into two lessons. The first can introduce how to access the `:ticker` route parameter in the callback and declare the `ticker` variable. The next lesson can ask the user to declare the `shares` variable.